corehq/apps/data_dictionary/util.py

```diff
@@ -207,9 +207,13 @@ def save_case_property(name, case_type, domain=None, data_type=None,
     if not name:
         return gettext('Case property must have a name')
 
-    prop = CaseProperty.get_or_create(
-        name=name, case_type=case_type, domain=domain
-    )
+    try:
+        prop = CaseProperty.get_or_create(
+            name=name, case_type=case_type, domain=domain
+        )
+    except ValueError as e:
+        return gettext(str(e))
```

Comment:
Is the exception going to be translated?

Reply:
After reading up a bit it seems like a bad idea to use `gettext` for use-cases like this. If translation is necessary, `ugettext_lazy` might be better, but the question is whether or not the exception _should_ be translated, in which case I think it might not be necessary (if fact, later in that same method we return an error without translating it, so maybe it's OK to not do that).